browser/brave_wayback_machine/brave_wayback_machine_util.cc

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_wayback_machine/brave_wayback_machine_util.h"
+
+#include "base/containers/flat_set.h"
+#include "brave/browser/brave_wayback_machine/brave_wayback_machine_infobar_delegate.h"
+#include "brave/common/pref_names.h"
+#include "chrome/browser/profiles/profile.h"
+#include "components/prefs/pref_service.h"
+#include "content/public/browser/web_contents.h"
+
+void CheckWaybackMachineIfNeeded(content::WebContents* contents,
+                                 int response_code) {
+  static base::flat_set<int> responses =
+      { 404, 408, 410, 451, 500, 502, 503, 504,
```

Comment:
can we use net when available? `net::HTTP_NOT_FOUND`

Reply:
also can you put these each on their own line so we can add a comment for the status code type if it's not in `net`?

Reply:
Done.